.github/labels.yml

```diff
@@ -0,0 +1,277 @@
+  name: TSC
+- color: f58220
+  description: Items related to Technical Steering Committee voting
+  name: TSC:voting
+
+# ##### type labels ##### #
+- color: b60205
+  description: ""
+  name: breaking change
+- color: d73a4a # red (standard github)
+  description: Something isn't working
+  name: bug
+- color: d73a4a
+  description: ""
+  name: bug fix
+- color: e5b479
+  description: ""
+  name: deprecation
+- color: a2eeef # light teal (standard github)
+  description: New feature or request
+  name: enhancement
+- color: 207de5
+  description: ""
+  name: feature
+- color: bfd4f2
+  description: ""
+  name: proposal
+- color: d876e3 # pink (standard github)
+  description: Further information is requested
+  name: question
+- color: fbca04
+  description: ""
+  name: refactor
+- color: d93f0b
+  description: ""
+  name: regression
+
+# ##### flag labels ##### #
+- color: 7057ff # purple (standard github)
+  description: Good for newcomers
+  name: good first issue
+- color: 008672 # teal (standard github)
+  description: Extra attention is needed
+  name: help wanted
+- color: ffffff
+  description: ""
+  name: stale
+- color: bfdadc
+  description: ""
+  name: WIP
+
+# ##### status labels ##### #
+- color: cfd3d7 # gray (standard github)
+  description: This issue or pull request already exists
+  name: duplicate # status:duplicate
+- color: e4e669 # yellow (standard github)
+  description: This doesn't seem right
+  name: invalid # status:invalid
+- color: ffffff # white (standard github)
+  description: This will not be worked on
+  name: wontfix # status:wontfix
+- color: C0C0C0
+  description: ""
+  name: status:can't reproduce
+- color: 7b5804
+  description: ""
+  name: status:need more info
+- color: E99695
+  description: ""
+  name: status:needs changes
+- color: D4C5F9
+  description: A manual tag to note PRs that are otherwise ready, but the CLA bot says the CLA has not been signed.
+  name: status:needs cla
+- color: d93f0b
+  description: ""
+  name: status:needs tests
+- color: cccccc
+  description: ""
+  name: status:not an issue
+- color: cccccc
+  description: Feature, design or idea with undecided status to include in the project functionality
+  name: status:on hold
+- color: fbca04
+  description: ""
+  name: status:to be verified
+- color: f7c6c7
+  description: ""
+  name: status:under discussion
+- color: 8FFCDE
+  name: "status: potential performance implications"
+  description: ""
+
+# ##### area/misc labels ##### #
+- color: FFFF5F
+  description: ""
+  name: external dependency
+- color: 2C5B58
+  description: ""
+  name: 'infrastructure: ci/cd'
+- color: 5F5F5F
+  description: ""
+  name: maintenance
+- color: d7e102
+  description: ""
+  name: tests
+
+# ##### area/code_location labels ##### #
+- color: 0075ca # blue (standard github)
+  description: Improvements or additions to documentation
+  name: documentation
+- color: 0075ca # blue
+  description: ""
+  name: API
```

Comment:
I think we can use this color of blue for `area/code_location` and `area/python` labels. Here is how `#0075ca` looks in dark and light mode:
![image](https://user-images.githubusercontent.com/1558590/144344700-af1c1466-2f35-4101-a434-496106ecbb47.png)